src/Microsoft.AspNet.Hosting.Interfaces/HostingEnvironmentExtensions.cs

```diff
@@ -40,6 +40,7 @@ public static class HostingEnvironmentExtensions
                 return hostingEnvironment.WebRootPath;
             }
 
+            virtualPath = virtualPath.Replace(Path.AltDirectorySeparatorChar, Path.DirectorySeparatorChar);
```

Comment:
Just weird is all. If we call out the behavior explicitly that would be better. Rather than things working as implemented